doc/sphinx/Pacemaker_Explained/rules.rst

```diff
@@ -254,21 +254,27 @@ A ``date_expression`` element may optionally contain a ``date_spec`` or
    |               |    pair: start; date_expression                           |
    |               |                                                           |
    |               | A date/time conforming to the                             |
-   |               | `ISO8601 <https://en.wikipedia.org/wiki/ISO_8601>`_       |
+   |               | `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_      |
    |               | specification. May be used when ``operation`` is          |
    |               | ``in_range`` (in which case at least one of ``start`` or  |
    |               | ``end`` must be specified) or ``gt`` (in which case       |
    |               | ``start`` is required).                                   |
+   |               |                                                           |
+   |               | Parsing of ISO 8601 timezones in date/time strings is     |
+   |               | currently incomplete and should not be relied upon.       |
    +---------------+-----------------------------------------------------------+
    | end           | .. index::                                                |
    |               |    pair: end; date_expression                             |
    |               |                                                           |
    |               | A date/time conforming to the                             |
-   |               | `ISO8601 <https://en.wikipedia.org/wiki/ISO_8601>`_       |
+   |               | `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_      |
    |               | specification. May be used when ``operation`` is          |
    |               | ``in_range`` (in which case at least one of ``start`` or  |
    |               | ``end`` must be specified) or ``lt`` (in which case       |
    |               | ``end`` is required).                                     |
+   |               |                                                           |
+   |               | Parsing of ISO 8601 timezones in date/time strings is     |
+   |               | currently incomplete and should not be relied upon.       |
```

Comment:
I'd be totally fine with putting this into a note since it's duplicated in `start` and `end`. I put it here to try to ensure the reader sees it.